ajax/libs/slim-select/package.json

```diff
@@ -21,7 +21,7 @@
     {
       "basePath": "dist",
       "files": [
-        "index*"
```

Comment:
Do i need to keep the index* for older versions of this package?

Reply:
Nope